### PR TITLE
Extrapolate poses for ground truth relations.

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer/ground_truth/compute_relations_metrics_main.cc [hunk 1]
```diff
@@ -109,6 +109,21 @@ std::string StatisticsString(const std::vector<Error>& errors) {
...
 
+transform::Rigid3d LookupTransform(
+    const transform::TransformInterpolationBuffer&
+        transform_interpolation_buffer,
+    const common::Time time) {
+  const common::Time earliest_time =
+      transform_interpolation_buffer.earliest_time();
+  if (transform_interpolation_buffer.Has(time)) {
+    return transform_interpolation_buffer.Lookup(time);
+  } else if (earliest_time < time) {
```
Comment (wohe) [inline cartographer/ground_truth/compute_relations_metrics_main.cc]: `time < earliest_time` you meant?

Comment (cschuet) [reply cartographer/ground_truth/compute_relations_metrics_main.cc]: Haha. Fixed.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**